### PR TITLE
#26827 Add API and Factory method to count Action and SubActions in a…

### PR DESCRIPTION
**Pull request by freddyDOTCMS**
### Proposed Changes
* Create a method on API and Factory level to count Actions and SubActions Workflow

### CODE REVIEW

**Review by AndreyDotcms — APPROVED** [review]
I think it looks great, nice work!

**Review by jcastro-dotcms — APPROVED** [review]

**Review by victoralfaro-dotcms — APPROVED** [review]

File: dotCMS/src/main/java/com/dotmarketing/portlets/workflows/business/WorkflowAPIImpl.java [hunk 1]
```diff
@@ -4375,6 +4375,26 @@ public long countAllSchemasSteps(final User user) throws DotDataException, DotSe
 		return workFlowFactory.countAllSchemasSteps();
 	}
 
+	public long countAllSchemasActions(final User user) throws DotDataException, DotSecurityException {
```
Comment (jdotcms) [inline dotCMS/src/main/java/com/dotmarketing/portlets/workflows/business/WorkflowAPIImpl.java:4378]: CloseDBIfOpened

File: dotCMS/src/main/java/com/dotmarketing/portlets/workflows/business/WorkflowAPIImpl.java [hunk 2]
```diff
@@ -4375,6 +4375,26 @@ public long countAllSchemasSteps(final User user) throws DotDataException, DotSe
...
+		try {
+			this.isUserAllowToModifiedWorkflow(user);
+		} catch (WorkflowPortletAccessException | InvalidLicenseException e) {
+			throw new DotSecurityException(e.getMessage(), e);
+		}
+
+		return workFlowFactory.countAllSchemasActions();
+	}
+
+	public long countAllSchemasSubActions(final User user) throws DotDataException, DotSecurityException {
```
Comment (jdotcms) [inline dotCMS/src/main/java/com/dotmarketing/portlets/workflows/business/WorkflowAPIImpl.java:4388]: CloseDBIfOpened

File: dotCMS/src/main/java/com/dotmarketing/portlets/workflows/business/WorkflowAPIImpl.java (hunk 1 above)
Comment (freddyDOTCMS) [reply dotCMS/src/main/java/com/dotmarketing/portlets/workflows/business/WorkflowAPIImpl.java:4378]: done https://github.com/dotCMS/core/pull/26856/commits/99eb040a84ccdc53d807e3583378cc275f4c3fbd#diff-9cbe979f949605cb72c988900af572a353ba138a6ddb53f257bb8f04e98d5606R496

File: dotCMS/src/main/java/com/dotmarketing/portlets/workflows/business/WorkflowAPIImpl.java (hunk 2 above)
Comment (freddyDOTCMS) [reply dotCMS/src/main/java/com/dotmarketing/portlets/workflows/business/WorkflowAPIImpl.java:4388]: done https://github.com/dotCMS/core/pull/26856/commits/99eb040a84ccdc53d807e3583378cc275f4c3fbd#diff-9cbe979f949605cb72c988900af572a353ba138a6ddb53f257bb8f04e98d5606R512

**Review by jdotcms — APPROVED** [review]

**Pull request merged**